lib/model/content.dart

```diff
@@ -1024,6 +1083,10 @@ class _ZulipContentParser {
       return parseImageNode(element);
     }
 
+    if (localName == 'div' && classes.containsAll(['message_inline_image', 'message_inline_video'])) {
```

Comment:
We'll want to use `className` rather than `classes`; we made that switch earlier this year. See `git log -p -S classes lib/model/content.dart` for examples and motivation.

This one will be a bit nontrivial (because there's more than one class), but see the latest couple of commits matched by that filter for examples of similar complexity.